spec/light-client/supervisor/supervisor_001_draft.md

```diff
@@ -463,69 +463,93 @@ func Sequential-Supervisor (initdata LCInitData) (Error) {
 #### Initialization
 
 The light client is based on subjective initialization. It has to
-trust the initial data given to it by the user. It cannot do any
-detection of attack. So either upon initialization we obtain a
-lightblock and just initialize the lightstore with it. Or in case of a
-genesis file, we download, verify, and cross-check the first block, to
-initialize the lightstore with this first block. The reason is that
-we want to maintain [LCV-INV-TP.1] from the beginning.
-
-> If the lightclient is initialized with a lightblock, one might think
-> it may increase trust, when one cross-checks the initial light
-> block. However, if a peer provides a conflicting
-> lightblock, the question is to distinguish the case of a
-> [bogus](https://informal.systems) block (upon which operation should proceed) from a
-> [light client attack](https://informal.systems) (upon which operation should stop). In
-> case of a bogus block, the lightclient might be forced to do
-> backwards verification until the blocks are out of the trusting
-> period, to make sure no previous validator set could have generated
-> the bogus block, which effectively opens up a DoS attack on the lightclient
-> without adding effective robustness.
+trust the initial data given to it by the user. It cannot perform any
+detection of an attack yet instead requires an initial point of trust.
+There are three forms of initial data:
+
+- A trusted block from a prior initialization
+- A trusted height and hash
+- A genesis file
+
+The golang light client implementation checks this initial data in that 
+order; first attempting to find a trusted block from the trusted store, 
+then acquiring a light block from the primart at the trusted height and matching
+the hash, or finally checking for a genesis file to verify the initial header.
+
+The light client doesn't need to check if the trusted block is within the
+trusted period because it already trusts it, however, if the light block is 
+outside the trust period, there is a higher chance the light client won't be
+able to verify anything. 
+
+Cross-checking this trusted block with providers upon initialization is helpful
+for ensuring liveness and correct configuration of such nodes but does not 
+increase trust since proving a conflicting block is a 
+[light client attack](https://informal.systems) 
+and not just a [bogus](https://informal.systems) block could result in
+performing backwards verification beyond the trusted period, thus a fruitless
+endeavour.
```

Comment:
Here I do note that cross-checking does no increase trust. I then go on to say that the `golang light client imlementation` performs a "consistency check" on all providers

Reply:
Sorry, I didn't see that earlier.

I am not sure I understand how it is helpful for liveness, though? Perhaps you could add a sentence that clarifies the scenario you have in mind?

Reply:
By liveness I'm meaning that we were able to establish a connection with the provider and they were responsive. This might actually be a different definition of liveness to the one you have in mind

Reply:
I didn't have any definition in mind ;-)  
I think if there is a specific goal, it should be written more explicitly.